aurorastation.dme

```diff
@@ -10,6 +10,7 @@
 #define DEBUG
 // END_PREFERENCES
 // BEGIN_INCLUDE
+#include "_maps\runtime.dm"
```

Comment:
Is there a reason the _maps\runtime.dm is included?

Reply:
Oops, forgot to remove that.